src/commands/prune.rs

```diff
@@ -589,9 +589,9 @@ impl Pruner {
         };
 
         self.repack_candidates.sort_unstable_by_key(|rc| rc.0);
-        let mut resize_packs: BlobTypeMap<Vec<_>> = Default::default();
-        let mut do_repack: BlobTypeMap<bool> = Default::default();
-        let mut repack_size: BlobTypeMap<u64> = Default::default();
+        let mut resize_packs: BlobTypeMap<Vec<_>> = BlobTypeMap::default();
+        let mut do_repack: BlobTypeMap<bool> = BlobTypeMap::default();
+        let mut repack_size: BlobTypeMap<u64> = BlobTypeMap::default();
```

Comment:
Can't we remove the type hint here?

Reply:
Unfortunately, only `do_repack` can automatically be inferred as BlobTypeMap<bool>, leaving off the type hints for `resize_packs` and `repack_size` leads to a compile error that rust can't infer the type.

1. Would you prefer leaving off the type hint for do_repack?
2. We could use e.g. `BlobTypeMap::<Vec<_>>::default()`, which do you prefer?

Reply:
I removed all type hints, using your second option where needed